### PR TITLE
Add german translations for version 1.1.0

### PR DESCRIPTION
**Pull request by holistic-developer**

### CODE REVIEW

**Comment by holistic-developer** [comment]
@Art4 Could you review this, since you did version 1.0.0 as well?

File: source/de/1.1.0/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+    schreiben, aber es völlig unterschiedlich aussprechen. <code>2012-06-02</code>
+    folgt der Logik vom größten zum kleinsten Wert, kann nicht mit anderen Formaten
+    verwechselt werden und ist ein #{link_to "ISO Standard", data.links.isodate}. Deshalb
+    ist es das empfohlene Datumsformat in Changelogs.
+
+  %h4#inconsistent-changes
+    %a.anchor{ href: "#inconsistent-changes", aria_hidden: "true" }
+    Inkonsistente Änderungen
+  %p
+    Ein Changelog das nur einen Teil der Änderung beinhaltet kann genauso gefährlich sein,
```
Comment (Art4) [inline source/de/1.1.0/index.html.haml:173]:
```suggestion
    Ein Changelog, das nur einen Teil der Änderung beinhaltet, kann genauso gefährlich sein,
```

File: source/de/1.1.0/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+    wie gar kein Changelog zu führen. Auch wenn manche Änderung nicht relevant sind - zum
```
Comment (Art4) [inline source/de/1.1.0/index.html.haml:174]:
```suggestion
    wie gar kein Changelog zu führen. Auch wenn manche Änderung nicht relevant sein mögen - zum
```

File: source/de/1.1.0/index.html.haml [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+    Beispiel muss das Entfernen von einzelnen Leerzeichen nicht in jedem Fall
+    protokolliert werden - sollen alle wichtigen Änderung im Changelog erwähnt werden.
```
Comment (Art4) [inline source/de/1.1.0/index.html.haml:176]:
```suggestion
    protokolliert werden - sollten alle wichtigen Änderung im Changelog erwähnt werden.
```

File: source/de/1.1.0/index.html.haml [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+    Werden Änderungen nur inkonsistent aufgezeichnet, könnten Benutzer fälschlicherweise
+    zu dem Schluss kommen, das Changelog sei die einzige Quelle der Wahrheit. Das sollte
```
Comment (Art4) [inline source/de/1.1.0/index.html.haml:178]:
```suggestion
    zu dem Schluss kommen, dass das Changelog die einzige Quelle der Wahrheit sei. Das sollte
```

File: source/de/1.1.0/index.html.haml [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+    verwechselt werden und ist ein #{link_to "ISO Standard", data.links.isodate}. Deshalb
+    ist es das empfohlene Datumsformat in Changelogs.
+
+  %h4#inconsistent-changes
+    %a.anchor{ href: "#inconsistent-changes", aria_hidden: "true" }
+    Inkonsistente Änderungen
+  %p
+    Ein Changelog das nur einen Teil der Änderung beinhaltet kann genauso gefährlich sein,
+    wie gar kein Changelog zu führen. Auch wenn manche Änderung nicht relevant sind - zum
+    Beispiel muss das Entfernen von einzelnen Leerzeichen nicht in jedem Fall
```
Comment (Art4) [inline source/de/1.1.0/index.html.haml:175]:
```suggestion
    Beispiel muss das Entfernen eines einzelnen Leerzeichens nicht in jedem Fall
```

File: source/de/1.1.0/index.html.haml [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+    protokolliert werden - sollen alle wichtigen Änderung im Changelog erwähnt werden.
+    Werden Änderungen nur inkonsistent aufgezeichnet, könnten Benutzer fälschlicherweise
+    zu dem Schluss kommen, das Changelog sei die einzige Quelle der Wahrheit. Das sollte
+    es auch sein. Aus großer Kraft folgt große Verantwortung - einen guten Changelog zu
```
Comment (Art4) [inline source/de/1.1.0/index.html.haml:179]:
```suggestion
    es aber sein. Aus großer Kraft folgt große Verantwortung - einen guten Changelog zu
```

File: source/de/1.1.0/index.html.haml [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+    besitzen heißt auch ihn konsistent zu aktualisieren.
```
Comment (Art4) [inline source/de/1.1.0/index.html.haml:180]:
```suggestion
    besitzen heißt, ihn konsistent zu aktualisieren.
```

**Review by Art4 — CHANGES_REQUESTED** [review]

File: source/de/1.1.0/index.html.haml [hunk 8]
```diff
@@ -0,0 +1,320 @@
+---
+description: Keep a Changelog
+title: Keep a Changelog
+language: de
+version: 1.0.0
```
Comment (Art4) [inline source/de/1.1.0/index.html.haml:5]:
```suggestion
version: 1.1.0
```

File: source/de/1.2.0/index.html.haml [hunk 9]
```diff
@@ -0,0 +1,320 @@
+---
+description: Keep a Changelog
+title: Keep a Changelog
+language: de
+version: 1.0.0
```
Comment (Art4) [inline source/de/1.2.0/index.html.haml:5]:
```suggestion
version: 1.2.0
```

**Review by Art4 — CHANGES_REQUESTED** [review]

**Comment by Art4** [comment]
Can someone please tell me where I can find the original text for version 1.2.0?

**Comment by holistic-developer** [comment]
Thanks a lot for the great review, @Art4!
The original text for version 1.2.0 can be found here
https://github.com/olivierlacan/keep-a-changelog/pull/370/commits/11876fd6a57de1977b739fa7021a3a7f39082974

I could also remove version 1.2.0 from this PR again and move it into a separate one to be merged when the English version is merged.

**Comment by Art4** [comment]
I would appreciate a separate PR for 1.2.0.

**Review by Art4 — APPROVED** [review]

**Pull request merged**